grabbit/src/main/groovy/com/twcable/grabbit/servlets/GrabbitServlet.groovy

```diff
@@ -56,15 +56,14 @@ class GrabbitServlet extends SlingAllMethodsServlet {
 
         if (path) { //Server Get Request
             final decodedPath = URLDecoder.decode(path, "utf-8")
+            final decodedExcludePaths = URLDecoder.decode(request.getParameter("excludePaths") ?: "", "utf-8")
```

Comment:
Instead of worrying about using special delimiters for sending between client and server, just use what's built into the servlet spec: [`getParameterValues(..)`](https://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html#getParameterValues(java.lang.String))

That means that you can pass around `Collection<String>` through the layers of the API instead of "a "`String`" but it's actually a collection of strings that hasn't been separated yet".

We'll still need to use a delimiter for the Batch Job parameters, but if it's later discovered that the chosen delimiter isn't appropriate, then it's an internal detail/fix rather than a part of the communication protocol between client and server. (Remember, ideally client and server should be able to evolve independently so that **_every_** upper environment doesn't have to change at exactly the same rate as **_every_** lower environment.)